.ci/set-up-globals.sh

```diff
@@ -17,18 +17,15 @@ touch $BASH_ENV
 ) >> $BASH_ENV
 source $BASH_ENV
 
-set -ex
-
-TERMINUS_PLUGINS_DIR=.. terminus list -n remote
```

Comment:
Shouldn't we just remove Circle now that we have GitHub actions?

Reply:
This is still being used by GitHub Actions.  You just commented where it's being used. :)

Reply:
Misread as `.circleci/...` 😊